gallery/src/pageConfig.ts

```diff
@@ -140,119 +133,124 @@ import NxStatefulCollapsibleRadioSelectPage
-    NxSearchTransferList: NxSearchTransferListPage,
-    NxStatefulSearchTransferList: NxStatefulSearchTransferListPage,
-    NxSubmitMask: NxSubmitMaskPage,
-    NxStatefulSubmitMask: NxStatefulSubmitMaskPage,
-    NxTable: NxTablePage,
-    NxTabs: NxTabsPage,
-    NxStatefulTabs: NxStatefulTabsPage,
-    NxTag: NxTagPage,
-    NxTextInput: NxTextInputPage,
-    NxStatefulTextInput: NxStatefulTextInputPage,
-    NxTextLink: NxTextLinkPage,
-    NxThreatCounter: NxThreatCounterPage,
-    NxSmallThreatCounter: NxSmallThreatCounterPage,
-    NxThreatIndicator: NxThreatIndicatorPage,
-    NxThreatIndicatorLegend: NxThreatIndicatorLegendPage,
-    NxToggle: NxTogglePage,
-    NxStatefulToggle: NxStatefulTogglePage,
-    NxTooltip: NxTooltipPage,
-    NxOverflowTooltip: NxOverflowTooltipPage,
-    NxTransferList: NxTransferListPage,
-    NxStatefulTransferList: NxStatefulTransferListPage,
-    NxTree: NxTreePage,
-    NxTreeView: NxTreeViewPage,
-    NxTreeViewMultiSelect: NxTreeViewMultiSelect,
-    NxStatefulTreeViewMultiSelect: NxStatefulTreeViewMultiSelect,
-    NxTreeViewRadioSelect: NxTreeViewRadioSelectPage,
-    NxStatefulTreeViewRadioSelect: NxStatefulTreeViewRadioSelectPage,
-    NxVulnerabilityDetails: NxVulnerabilityDetailsPage
+  'Alerts, Indicators, and Notifications': {
+    Alert: { content: NxAlertComponentsPage, type: 'react' },
+    StatefulAlert: { content: NxStatefulAlertComponentsPage, type: 'react' },
+    BinaryDonutChart: { content: NxBinaryDonutChartPage, type: 'react' },
+    'counter': { content: NxCounterPage, type: 'html' },
+    FontAwesomeIcon: { content: NxFontAwesomeIconPage, type: 'react' },
+    LoadError: { content: NxLoadErrorPage, type: 'react' },
+    LoadWrapper: { content: NxLoadWrapperPage, type: 'react' },
+    LoadingSpinner: { content: NxLoadingSpinnerPage, type: 'react' },
+    PolicyViolationIndicator: { content: NxPolicyViolationIndicatorPage, type: 'react' },
+    SubmitMask: { content: NxSubmitMaskPage, type: 'react' },
+    StatefulSubmitMask: { content: NxStatefulSubmitMaskPage, type: 'react' },
+    Tag: { content: NxTagPage, type: 'react' },
+    ThreatCounter: { content: NxThreatCounterPage, type: 'react' },
+    SmallThreatCounter: { content: NxSmallThreatCounterPage, type: 'react' },
+    ThreatIndicator: { content: NxThreatIndicatorPage, type: 'react' },
+    ThreatIndicatorLegend: { content: NxThreatIndicatorLegendPage, type: 'react' },
+    'threat-number': { content: NxThreatNumberPage, type: 'html' },
+    Tooltip: { content: NxTooltipPage, type: 'react' },
+    OverflowTooltip: { content: NxOverflowTooltipPage, type: 'react' }
   },
-  'Guidelines': {
-    'Accessibility': AccessibilityPage,
-    'Additional Resources': AdditionalResourcePage,
-    'Contributing to the Gallery': ContributingPage,
-    'font-size and line-height': FontSizePage,
-    'Form Validation Guidelines': FormValidationPage,
-    'Styling Components': StylingComponentsPage
+  'Buttons and Dropdowns': {
```

Comment:
A handful of things in this sections still have the `Nx` prefix

Reply:
> A handful of things in this sections still have the `Nx` prefix

Fixed here:https://github.com/sonatype/sonatype-react-shared-components/pull/596/commits/983e7ab27c49293f2b44d72e8a4b3088f0e28f71